store/postgres.go

```diff
@@ -178,7 +179,10 @@ func (s *PGXStore) ListActors(ctx context.Context, opts ListActorsOpts) (out []*
 		statusFilter.ActorStatus = status
 	}
 
-	actors, err := s.queries.ListCandidateActors(ctx, statusFilter)
+	actors, err := s.queries.ListCandidateActors(ctx, gen.ListCandidateActorsParams{
+		Status: statusFilter,
+		Limit:  opts.Limit,
```

Comment:
This change breaks the CandidateActorCache as that hasn't been updated to pass a limit in - so 0 will be used. We may need to build pagination into that before we can do this 😓